packages/terra-functional-testing/tests/jest/commands/utils/ScreenshotRequestor.test.js

```diff
@@ -142,6 +149,160 @@ describe('ScreenshotRequestor', () => {
     });
   });
 
+  describe('downloadScreenshots', () => {
```

Comment:
Can we make this more clear by changing this to:
```suggestion
  describe('function downloadScreenshots', () => {
```

Reply:
Changed Here: https://github.com/cerner/terra-toolkit/pull/808/commits/8e94641850bdce7c1c53eca331d2198f71d1e3b5#diff-d5e5e839b475dfb26a79344066ab8d867df7550d65a2eed690e281a9d9a0946cR152